pkg/config/debug.go

```diff
@@ -61,6 +62,16 @@ func (c *DebugConfig) ValidateAndAdjust() error {
 	if err := c.Scheduler.ValidateAndAdjust(); err != nil {
 		return errors.Trace(err)
 	}
+	if c.Scheduler.RegionPerSpan != 0 {
+		if c.EnablePullBasedSink || !c.EnableNewSink {
```

Comment:
The `debug.enable-pull-based-sink` default is true. Does this mean you want to implement it in the table pipeline first?

Reply:
Yes, for now, I think table pipeline is more stable for testing this feature.